src/components/get_started/Contribute.jsx

```diff
@@ -1,12 +1,15 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
```

Comment:
remove

Reply:
I'm confused, your commit adds these...and this comment says to remove them?

Reply:
Yeh. This is a "test" commit. It still had these redundant lines in them. I wanted you to see the problem. The only problem is the lines where I explicitly say "This does not work"